packages/patternfly-4/react-core/src/components/Nav/NavListItem.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Nav/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Content rendered inside the list item */
+  children: PropTypes.node.isRequired,
+  /** Additional classes added to the list item */
+  className: PropTypes.string,
+  /** Target navigation link */
+  to: PropTypes.string,
+  /** Flag indicating whether the item is active */
+  isActive: PropTypes.bool
+};
+
+const defaultProps = {
+  className: '',
+  to: '#',
+  isActive: false
+};
+
+/* eslint react/prop-types: 0 */
+class NavListItem extends React.Component {
+  onUpdate(e) {
+    const { onSelect, groupId, id } = this.props;
+    if (e) {
```

Comment:
No need to check for the event to be present, but if you really want to be defensive this can be simpliified with shortcircuit
```JS
e && e.stopPropagation();
```